qemu/tests/cfg/steal_time.cfg

```diff
@@ -0,0 +1,16 @@
+- steal_time:
+    type = steal_time
+    virt_test_type = qemu
+    only Linux
+    only x86_64, i386
+    kill_vm = yes
+    vms += " vm0"
+    image_snapshot = yes
+    qemu_command_prefix = "taskset -c 1 "
+    smp = 1
+    vcpu_maxcpus = 1
+    vcpu_sockets = 1
+    vcpu_threads = 1
+    stress_args = --cpu 1 --vm 2 --vm-bytes 1024 
+    top_cmd = top -n 1 | grep Cpu
+    stat_cmd = cat /proc/stat | head -1
```

Comment:
Use the sum of all cpus ? No the one be used in taskset ?

Reply:
It will run inside guest, should have nothing to do with taskset.